packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -312,7 +312,10 @@ const INDIVIDUAL_ACTIONS = [
     level: 'primary',
   },
   {
-    handler: (remote, { editRemote }) => editRemote(remote),
+    handler: (remote, { reset, editRemote }) => {
+      reset()
+      editRemote(remote)
```

Comment:
Don't ignore the promise.

Reply:
Idem, `editRemote` isn't async here since it's not the call to `xo-server`.